OrbitCore/LibunwindstackUnwinder.cpp

```diff
@@ -0,0 +1,68 @@
+#include "LibunwindstackUnwinder.h"
+
+#include "PrintVar.h"
+
+bool LibunwindstackUnwinder::SetMaps(const std::string& maps_buffer) {
+  maps_ = std::make_unique<unwindstack::BufferMaps>(maps_buffer.c_str());
+
+  if (!maps_->Parse()) {
+    PRINT("LibunwindstackUnwinder::SetMaps: !maps_->Parse()\n");
+    maps_ = nullptr;
+    return false;
+  }
+
+  return true;
+}
+
+const std::array<int, unwindstack::X86_64_REG_LAST>
+    LibunwindstackUnwinder::UNWINDSTACK_REGS_TO_PERF_REGS{
+    PERF_REG_X86_AX,
+    PERF_REG_X86_DX,
+    PERF_REG_X86_CX,
+    PERF_REG_X86_BX,
+    PERF_REG_X86_SI,
+    PERF_REG_X86_DI,
+    PERF_REG_X86_BP,
+    PERF_REG_X86_SP,
+    PERF_REG_X86_R8,
+    PERF_REG_X86_R9,
+    PERF_REG_X86_R10,
+    PERF_REG_X86_R11,
+    PERF_REG_X86_R12,
+    PERF_REG_X86_R13,
+    PERF_REG_X86_R14,
+    PERF_REG_X86_R15,
+    PERF_REG_X86_IP,
+};
+
+std::vector<unwindstack::FrameData> LibunwindstackUnwinder::Unwind(
+    const std::array<uint64_t, PERF_REG_X86_64_MAX>& perf_regs,
+    const char* stack_dump, uint64_t stack_dump_size) {
+
+  if (!maps_) {
```

Comment:
Is this an error case or a legitimate case?

Reply:
It's an error, added a message.